### PR TITLE
add link to family sample in sample and family views

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
This PR add link to family sample in sample and family views

**How to test Family link to FamilySample**:
1. install on stage of the clinical-db machine: `bash servers/resources/clinical-db.scilifelab.se/update-clinical-api-stage.sh link-family-and-sample-to-familysample`
1. login to clinical-api-stage
1. go to family view
1. click the link in the column internal id in a family

**Expected outcome**:
The familysample view is shown for the family that was chosen

**How to test Sample link to FamilySample**:
1. go to sample view
1. click the link in the column internal id in a sample

**Expected outcome**:
The familysample view is shown for the sample that was chosen

**Review:**
- [x] code approved by @ingkebil 
- [x] tests executed by @patrikgrenfeldt 
- [x] "Merge and deploy" approved by @ingkebil 
Thanks for filling in who performed the code review and the test!

This is minor **version bump** because it adds functionality in a backwards compatible manner

### CODE REVIEW

File: cg/server/admin.py [hunk 1]
```diff
@@ -17,6 +17,18 @@ def inaccessible_callback(self, name, **kwargs):
         return redirect(url_for('google.login', next=request.url))
 
 
+def view_family_sample_link(unused1, unused2, model, unused3):
```
Comment (ingkebil) [inline cg/server/admin.py]: I guess the `unsused` ones are params that are passed by default?

**Comment by ingkebil** [comment]
Code looks good!

File: cg/server/admin.py (hunk 1 above)
Comment (patrikgrenfeldt) [reply cg/server/admin.py]: yes, I'm leaning towards another way of doing it, to use underscore (_, _, model, _), but then we can't delete them like we do now, what's your preference?

**Comment by patrikgrenfeldt** [comment]
Test outcome: 
Family view with one searched family: 
image

Link in family view leads to FamilySample view:
image

Sample view with one searched sample:
image

Link in sample view leads to FamilySample view:
image

File: cg/server/admin.py (hunk 1 above)
Comment (ingkebil) [reply cg/server/admin.py]: The latter definitely is how I've seen everyone else dealing with unused and unneeded variables. So let's go for that

**Comment by ingkebil** [comment]
I'm approving pending the requested change

File: cg/server/admin.py (hunk 1 above)
Comment (patrikgrenfeldt) [reply cg/server/admin.py]: It turns out that if you use the underscore more than once in the signature then it's considered to be a name collision...

**Pull request merged**